src/corehost/cli/comhost/comhost.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
We have stopped using this license header in 2016. Looks like `core-setup` was missed during the license header cleanup.

Reply:
I filed #5224 for that.